static/js/lib/ckeditor/plugins/validateMdConversion.ts

```diff
@@ -0,0 +1,29 @@
+import * as Sentry from "@sentry/react"
+import { isEqual } from "lodash"
+const domParser = new DOMParser()
+
+const gatherDocumentStats = (doc: Document) => {
+  const tables = doc.querySelectorAll("table")
+  return {
+    tables: tables.length
+  }
+}
+
+const validateHtml2md = (
+  md: string,
+  html: string,
+  md2html: (md: string) => string
+): void => {
+  const htmlDoc = domParser.parseFromString(html, "text/html")
+  const mdDoc = domParser.parseFromString(md2html(md), "text/html")
```

Comment:
This could conceivably have a noticeable performance hit on our CKEditor instance: we are now calling `md2html` twice per edit. It's possible that's noticeable when many keystrokes happen in quick succession.

If that happens, we should debounce the `onChange` handlers.